extension-style/src/main/java/com/mapbox/maps/extension/style/layers/LayerExt.kt

```diff
@@ -145,4 +149,6 @@ fun StyleInterface.addPersistentLayer(layer: Layer, position: LayerPosition? = n
 @MapboxExperimental
 fun Layer.isPersistent(): Boolean? {
   return delegate?.isStyleLayerPersistent(layerId)?.value
-}
\ No newline at end of file
+}
+
+const val TAG = "Mbgl-LayerUtils"
```

Comment:
could be made private? We might be leaking this tag if not?

Reply:
`getLayerAs` is a public inline function, which cannot use Private / internal declarations.
Although I can use @PublishedApi annotation to make TAG internal and let the function use it. but basically, that would be public TAG in a way.

Reply:
Fixed by annotating TAG with @PublishedApi in order to use it in public inline function `getLayerAs`.